c/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.24 FATAL_ERROR)
 
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
Are these necessary? The modern way would be target_compile_features as already used here: https://github.com/tira-io/tirex-tracker/blob/abfc54c52b956e2d6e26805f71ce463640aaa402/c/src/CMakeLists.txt#L20
Currently, I see two issues with setting CXX_STANDARD and requiring it:
1. It prohibits later standards
2. It is global instead of per target

Reply:
Good catch 👍 Actually, I have no clue but just found that online as a workaround. I'll try without.